tests/e2e/pageobjects/ide/Ide.ts

```diff
@@ -22,7 +22,7 @@ export enum LeftToolbarButton {
     Explorer = 'Explorer',
     Git = 'Git',
     Debug = 'Debug',
-    Openshift = 'OpenShift'
+    Openshift = 'OpenShift',
```

Comment:
@makambalaji this doesn't look right. There are no more variables after this one, the `,` shouldn't be there.